styles/components/_misc.scss

```diff
@@ -91,9 +95,8 @@
 }
 
 .emoji {
-  height: 1em;
-  width: 1em;
-  margin-right: 0.3em;
+  height: 1em !important; // don't allow parent text sizes to override this
+  width: 1em !important;  // don't allow parent text sizes to override this
```

Comment:
This fixes some issues where the emoji would be off center if the parent text size was in ems.